roles/bibdata/defaults/main.yml

```diff
@@ -2,4 +2,4 @@
 running_on_server: false
 bibdata_samba_source_host: 'bibdata-worker1.princeton.edu'
 bibdata_mount_env: 'staging'
-bibdata_data_dir: '/data/marc_liberation_files'
```

Comment:
I think we'll want to keep the files in this directory on-hand and accessible.  Could we add something like this to the roles/bibdata/tasks/mounts.yml?

```
  - name: bibdata | Check if there are old files in /data/marc_liberation_files
    ansible.builtin.stat:
       path: /data/marc_liberation_files
       register: marc_liberation_files_dir
  - name: bibdata | Move over old marc_liberation_files
    ansible.builtin.copy:
      src: '/data/marc_liberation_files/*'
      dest: '{{ bibdata_data_dir }}'
      when: marc_liberation_files_dir.exists
  - name: bibdata | Remove old marc_liberation_files directory
    ansible.builtin.file:
       path: /data/marc_liberation_files
       state: absent
```

Reply:
@sandbergja thanks for catching this. Yes we definitely want to keep the files in this directory. I think It will take some time to move them. @kayiwa @acozine would it be better to create the new directory `bibdata_data_dir: '/data/bibdata_files` and copy the files there before we run this playbook ??

Reply:
I wonder if we could solve this by adding a line to the playbook that renames the directories before changing the config . . . that way we would not have to move the files. Alternatively, we could just keep the old directory names, but that feels like adding a layer of cruft.

Reply:
Since this is just a symlink, we think it will be okay. Testing on the staging servers first!